src/app/[locale]/_utils/mixpanel.ts

```diff
@@ -0,0 +1,97 @@
+import mixpanel from 'mixpanel-browser';
+
+const ANALYTICS_ENABLE = process.env.NEXT_PUBLIC_ANALYTICS_ENABLE;
+const ANALYTICS_MOCK = process.env.NEXT_PUBLIC_ANALYTICS_MOCK;
+const ANALYTICS_TOKEN = process.env.NEXT_PUBLIC_ANALYTICS_TOKEN || '';
+const ANALYTICS_API_HOST = process.env.NEXT_PUBLIC_ANALYTICS_API_HOST;
+const ANALYTICS_PERSISTENCE = process.env.NEXT_PUBLIC_ANALYTICS_PERSISTENCE;
+const ANALYTICS_LOG_IP = process.env.NEXT_PUBLIC_ANALYTICS_LOG_IP === 'true' ? true : false;
+const ANALYTICS_DEBUG = process.env.NEXT_PUBLIC_ANALYTICS_DEBUG === 'true' ? true : false;
+
+/** To call in order to start the analytics service, otherwise no event will be sent */
+export const initAnalytics = (): void => {
+  if (ANALYTICS_ENABLE) {
+    // eslint-disable-next-line functional/immutable-data, @typescript-eslint/no-explicit-any
+    (window as any).initMixPanel = true;
+    if (ANALYTICS_MOCK) {
```

Comment:
I see that analytics mock is set to `false` in our `.env*` files. Maybe should we consider to set it to `true` with a FIXME comment, to track to remove it once in production? Or is there another rationale beyond this choice?

Reply:
sure